CleanWeather/Scenes/CitiesList/Displayable/CitiesDisplayable.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CitiesDisplayable.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 12/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct CitiesDisplayable {
+    
+    let cityName: String
+    let cityWeatherIcon: String
+    let cityTempRounded: Int
+    
+    static func convert(from object: Weather) -> CitiesDisplayable {
```

Comment:
tu bym się zastanowił nad modelem Weather, czy nie lepiej mu nadać nazwe która będzie bardziej adekwatna

bo jeden Weather ma dane 1 City, wiec moze CityWeather?

i konwertujesz do CityWeatherDisplayable?

Reply:
To co napisałem w komentarzu do nazewnictwa metody fetchWeather odnosi się również do powyższego